docs/vsphere-golden-image.md

```diff
@@ -4,13 +4,16 @@ This guide describes the thought process of creating a Windows virtual machine b
 
 ## 1. Select a compatible Windows Server version
 
-Currently, the Windows Machine Config Operator (WMCO) stable version only supports Windows Server Semi-Annual
-Channel (SAC): Windows Server 2004, that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351), 
-required by the operating system to allow using the [hybrid OVN Kubernetes networking with a 
-custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere) feature.
+Currently, the Windows Machine Config Operator (WMCO) stable version supports Windows Server Long-Term Servicing Channel
+(LTSC): Windows Server 2022 and Windows Server Semi-Annual Channel (SAC): Windows Server 20H2.
+
+*Please note that Windows Server Semi-Annual Channel (SAC): Windows Server 2004 is no longer supported. 
+Also, Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809 cannot be used since patch 
+[KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351) is not available, which is
+required by the operating system to allow using the 
+[hybrid OVN Kubernetes networking with a custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere) feature.*
```

Comment:
As above LTSC is not important here, but some important info that its missing is that this is actually talking about Windows Server 2019.

This can be condensed to:
Please note that Windows Server 2004 is no longer supported. 
Also, Windows Server 2019 cannot be used since patch  [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351) is not available, which is required by the operating system to allow using the [hybrid OVN Kubernetes networking with a custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere) feature

Reply:
I thought the note was intentional with Windows Server 2019, version 1809 (which is still in service) since Windows Server 2019, 1909 version does have the linked patch (but is EOL).

I can make it a bit more explicit by saying `Windows Server 2019, version 1809 (LTSC)`.